demo/common/src/window.rs

```diff
@@ -114,7 +114,7 @@ pub struct OcularTransform {
 }
 
 #[derive(Clone)]
-pub enum SVGPath {
+pub enum DataPath {
```

Comment:
Maybe `FilePath`?

Reply:
Technically, it doesn't have to be a file… I could see URLs useful too.
Maybe `DataSource`?